code/src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import './Header.css'
+import { Link } from "react-router-dom"
+import facebookIcon from '../assets/facebook-icon.png'
+import twitterIcon from '../assets/twitter-icon.png'
+import youtubeIcon from '../assets/youtube-icon.png'
+
+const Header = () => {
+    return (
+        <section className="header-container">
+            <div>
+                <Link to='/'><p className="header-title">Movieflix</p></Link>
+            </div>
+            <div className="media-icons-container">
+                <img className="media-icons" src={facebookIcon} alt=" facebook-icon"></img>
```

Comment:
apparently it is unnecessary to write image or icon in the alt text. Something to think about for the future(I always forget :P)